app/models/models.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+
+
+class QuestionsModels:
+
+    def __init__(self,  title, body, tags, qtn_id):
+        self.qtn_id = qtn_id
```

Comment:
Still wondering why you opted to create an id basing on the length of the list and not use uuid like you did in the `answer_models.py`. Also this shows inconsistence in your code.

Reply:
this was for simplicity instead of having an id that is a bit complicated. but i can change that